hack/release.sh

```diff
@@ -31,8 +31,8 @@ readonly COMPONENTS
 function build_release() {
    # Update release labels if this is a tagged release
   if [[ -n "${TAG}" ]]; then
-    echo "Tagged release, updating release labels to natss.eventing.knative.dev/release: \"${TAG}\""
-    LABEL_YAML_CMD=(sed -e "s|natss.eventing.knative.dev/release: devel|natss.eventing.knative.dev/release: \"${TAG}\"|")
+    echo "Tagged release, updating release labels to nats.eventing.knative.dev/release: \"${TAG}\""
+    LABEL_YAML_CMD=(sed -e "s|nats.eventing.knative.dev/release: devel|nats.eventing.knative.dev/release: \"${TAG}\"|")
```

Comment:
You probably need to add something about replacing the natsjsm.eventing release labels too?